components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel.test.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {mountWithIntl} from 'tests/helpers/intl-test-helper';
+
+import EnterpriseEditionLeftPanel, {EnterpriseEditionProps} from './enterprise_edition_left_panel';
+
+describe('components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel', () => {
+    const license = {
+        IsLicensed: 'true',
+        IssuedAt: '1517714643650',
+        StartsAt: '1517714643650',
+        ExpiresAt: '1620335443650',
+        SkuShortName: 'Enterprise',
+        Name: 'LicenseName',
+        Company: 'Mattermost Inc.',
+        Users: '1000000',
+    };
+
+    const props = {
+        license,
+        openEELicenseModal: jest.fn(),
+        upgradedFromTE: false,
+        isTrialLicense: false,
+        issued: <></>,
+        startsAt: <></>,
+        expiresAt: <></>,
+        handleRemove: jest.fn(),
+        isDisabled: false,
+        removing: false,
+    } as EnterpriseEditionProps;
+
+    test('should format the Users field', () => {
+        const wrapper = mountWithIntl(
+            <EnterpriseEditionLeftPanel {...props}/>,
+        );
+
+        const item = wrapper.find('.item-element').filterWhere((n) => {
```

Comment:
Is there a better way to target the content here? Rendered HTML code looks like this:

```html
<div class="licenseElements">
    <div class="item-element">
        <span class="legend">START DATE:</span>
        <span class="value"><span>6/7/2021</span></span>
    </div>
[...]
    <div class="item-element">
        <span class="legend">USERS:</span>
        <span class="value"><span>200,000</span></span>
    </div>
[...]
</div>
```

I could not find a semantic way to tackle this

Reply:
well, this seem to work, but I agree that it's too elaborate for getting an element. If you like, you could check if something like `wrapper.find({children: "USERS:"]})` works or set a `data-testid` attribute to use it to target that element. If any of those work for you, what you have already set is ok.